clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/properties/CamundaClientProperties.java

```diff
@@ -175,6 +175,18 @@ public URI getGrpcAddress() {
   }
 
   public void setGrpcAddress(final URI grpcAddress) {
+    /*
```

Comment:
Move to Util class (CamundaClientConfigurationUtil?) in the java client and call the method (CamundaClientConfigurationUtil.validateGrpcAddress) from there.